src/develop/imageop.c

```diff
@@ -2339,7 +2341,7 @@ void add_remove_mask_indicator(dt_iop_module_t *module, gboolean add)
     if(raster)
       gtk_widget_set_tooltip_text(module->mask_indicator, _("this module has a raster mask"));
     else
-      gtk_widget_set_tooltip_text(module->mask_indicator, _("this module has a mask\nclick to display"));
+      gtk_widget_set_tooltip_text(module->mask_indicator, _("this module has a mask, click to display\nmodule must be activated first"));
```

Comment:
I'll merge this one, but as an enhancement we could display in the tooltip if we have a drawn, parametric or both kind of mask. Let me know if you have time to look at this, I could do that otherwise. Just let me know.

Reply:
I can do it, no pb